src/main/java/org/opentripplanner/graph_builder/module/osm/OpenStreetMapModule.java

```diff
@@ -846,6 +846,11 @@ private void buildElevatorEdges(Graph graph) {
             /* build elevator edges */
             for (Long nodeId : multiLevelNodes.keySet()) {
                 OSMNode node = osmdb.getNode(nodeId);
+
+                // Use the first elevator node as the elevator label suffix
+                // Prevent duplicated vertex for elevator using the sames stops (chained elevator)
+                String elevatorLabelSuffix = getNodeLabel(node);
```

Comment:
I think it might be better to use the OSM way id here, as it is possible to have multiple ways between the same two end nodes.

Reply:
This is a simple case of a 'virtual' elevator created for node (highway=elevator) with ways on different levels, where there's no 'elevator' way, like this one: https://www.openstreetmap.org/node/7856391980 . I guess indeed for this one a suffix is not necessary as this will only but executed once per node id (hashmap key).